src/test/ui/parser/issue-59418.rs

```diff
@@ -0,0 +1,18 @@
+struct X(i32,i32,i32);
+
+fn main() {
+    let a = X(1, 2, 3);
+    let b = a.1suffix;
+    //~^ ERROR suffixes on a tuple index are invalid
+    println!("{}", b);
+    let c = (1, 2, 3);
+    let d = c.1suffix;
+    //~^ ERROR suffixes on a tuple index are invalid
+    println!("{}", d);
+    let s = X { 0suffix: 0, 1: 1, 2: 2 };
```

Comment:
How about `X { 0usize: 0, 1: 1, 2: 2 };`?

Reply:
(don't r- the PR in either case since it's in a rollup now... if changes need to be made, do it in a follow up PR)